notebooks/238-dolly-2-instruction-following/238-dolly-2-instruction-following.ipynb

```diff
@@ -0,0 +1,483 @@
+{
```

Comment:
<p>Text is good, but I think we should split last paragraph, it is not easy to read. Perhaps one split point is after decoding and another one is before "The generation cycle"</p><p>You might also want to mention temperature and topk as settings for decoding, I think this is setting that is exposed as well.</p><p> </p>
<br/>

 _Reply via <a href='https://app.reviewnb.com/openvinotoolkit/openvino_notebooks/pull/1041/discussion/'>ReviewNB</a>_ <div id='ReviewNBCommentContext-DoNotDelete' style='display:none' data-state='OPEN' data-cellIndex='5'/>